### PR TITLE
Add aggregateSeriesLists() and aliases for diffSeriesLists(), sumSeriesLists(), multiplySeriesLists()

### PR DESCRIPTION
**Pull request by alikhtag**
Closes: #2645  see for more information. 

Decided to use `aggregate()` as base function when implementing `diffSeriesLists(), sumSeriesLists(), multiplySeriesLists()` as aliases using `aggregateSeriesLists()`. 
Side effect of course is that other functions that are supported by `aggregate()` are also supported by `aggregateSeriesLists()`.

### CODE REVIEW

**Review by deniszh — APPROVED** [review]

**Comment by deniszh** [comment]
Cool! Good work, @alikhtag !

**Comment by alikhtag** [comment]
Can this be merged in near future, would like to use `master` branch for our images :relaxed:

**Review by piotr1212 — APPROVED** [review]

**Comment by AlikhanTagybergen-TomTom** [comment]
Thanks for quick merge @piotr1212 ! Will probably try to add these functions to `grafana` tooltip hint, since even old `divideSeriesLists()` is missing.

**Comment by alikhtag** [comment]
Thanks for quick merge @piotr1212 ! Will probably try to add these functions to `grafana` tooltip hint, since even old `divideSeriesLists()` is missing.

**Comment by alikhtag** [comment]
Hi @piotr1212 and/or @deniszh , can docs be deployed with latest version of master please?

**Comment by deniszh** [comment]
Hi @alikhtag ,
Docs built on https://github.com/graphite-project/graphite-web/commit/cc51437d4aa987a6e9128783472375b17a9f0bd8, but functions are sill not visible :(

**Comment by deniszh** [comment]
Ah no, it's there, looks like I hit cached version.

**Comment by alikhtag** [comment]
Thanks a lot / Большое Спасибо))) :) @deniszh!

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -1039,6 +1117,38 @@ def multiplySeries(requestContext, *seriesLists):
 ]
 multiplySeries.aggregator = True
 
+def multiplySeriesLists(requestContext, seriesListFirstPos, seriesListSecondPos):
+  """
+  Iterates over a two lists and subtracts series lists 2 through n from series 1
```
Comment (jmuf) [inline webapp/graphite/render/functions.py:1133]: typo in docs: "subtracts.. from" should be "multiplies.. by"

**Pull request merged**